README.md

```diff
@@ -13,10 +13,15 @@ Maven for building Eclipse bundles and features.
 
 ## Requirements
 
-1. Eclipse 4.5 (Mars) or later.  It's easiest to use the _Eclipse IDE for Java EE Developers_.
+1. The [Google Cloud SDK](https://cloud.google.com/sdk/); install
+  this somewhere on your file system.
 
-  1. The [m2eclipse plugin](http://www.eclipse.org/m2e/) (also called m2e) installed
-     to import the projects into Eclipse.
+1. [Eclipse 4.5 (Mars) or later](https://www.eclipse.org/downloads/eclipse-packages/). 
+  It's easiest to use the _Eclipse IDE for Java EE Developers_.
+
+  1. The [m2eclipse plugin](http://www.eclipse.org/m2e/) (also called m2e) is
+     required to import the projects into Eclipse.  m2eclipse is bundled with 
+     the _Eclipse IDE for Java EE Developers_ package.
```

Comment:
FYI: Recently, I did fresh installations of _Eclipse IDE for Java Developers_ (not the EE version) Mars (4.5) a few times. I realized that it comes with the "Existing Maven Project" item in the import wizard. I could import the whole projects without installing m2e manually.